### DISCUSSION TITLE
Minutes Oct. 31, 2024

### DISCUSSION THREAD

**Discussion by jgroenen** [standard-for-public-code/standard-for-public-code#1050] [Community calls]
A small Haiku:

I (Johan) joined at 15
But nobody joined
Ended the call at '10 past

**Comment by jgroenen** [comment]
Next one stands at 2nd Christmas Day (Dec. 26), so I think we need to move that @clausmullie

**Reply by clausmullie** [reply]
Yes, but I don't have editing rights, so we need to create a new calendar invite

**Reply by clausmullie** [reply]
Did you register a domain in the end? And if so, which one? I can't quite remember

**Reply by clausmullie** [reply]
A, found it, and see it's already live

**Comment by jgroenen** [comment]
Also, the publiccode.net notebook functionality seems to have been taken down, so maybe we can use another hosted one: https://cryptpad.disroot.org/pad/#/2/pad/edit/UJRsmvC-kUdTfxyr-+AN-Uqr/
